### PR TITLE
Document the different template parameters.

### LINKED ISSUE #25: document every yaml param - descriptions en gros are missing
e.g. https://github.com/opendevstack/ods-core/blob/master/jenkins/ocp-config/bc.yml#L72

we should use the same description as we have in 
https://github.com/opendevstack/ods-configuration-sample/blob/master/ods-core/jenkins/ocp-config/bc.env.sample 

to have this really consistent.

### PR DESCRIPTION
**Pull request by rattermeyer**
parameters are documented somewhat already in the configuration
directories, but not directly in the templates.
This has been fixed now.

resolves #25

### CODE REVIEW

File: jenkins/ocp-config/bc.yml [hunk 1]
```diff
@@ -70,7 +70,10 @@ objects:
       type: Docker
     triggers: []
 parameters:
+# repository base url. In case bitbucket is used, needs to include /scm - so it's the path in front on opendevstack/..
```
Comment (michaelsauter) [inline jenkins/ocp-config/bc.yml]: You can add them directly into the YML by using the `description` key.

**Comment by michaelsauter** [comment]
FYI @renedupont

File: jenkins/ocp-config/bc.yml (hunk 1 above)
Comment (rattermeyer) [reply jenkins/ocp-config/bc.yml]: was not sure. we have both styles. but I would also prefer the description, unless we have multiple lines. So, I would propose: Include a (short) description and add further information as comments

Comment (michaelsauter) [reply jenkins/ocp-config/bc.yml]: Sounds good.

**Comment by rattermeyer** [comment]
@michaelsauter pushed the changed templates

**Pull request merged**